### DISCUSSION TITLE
Page break in PDF Templater not working

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16859] [How to] [answered]
Cannot move content to the next page in PDF (latest stable). Fixes provided to the same issue before not working here. Can you guys confirm the same?

**Comment by apcloic** [comment, marked as answer]
Hi,

It looks like page-break is properly working on my system (6.4 latest stable) and also on gitstable, I'm using the following in my PDF template.

````
<div style="page-break-after:always"></div>
$(record : RecordId)$
````
